examples/pa_engine_multiple_unit_example.py

```diff
@@ -0,0 +1,129 @@
+from fds.analyticsapi.engines.api_client import ApiClient
+from fds.analyticsapi.engines.configuration import Configuration
+from fds.analyticsapi.engines.model.component_summary import ComponentSummary
+from fds.analyticsapi.engines.model.pa_calculation_parameters_root import PACalculationParametersRoot
+from fds.analyticsapi.engines.model.pa_calculation_parameters import PACalculationParameters
+from fds.analyticsapi.engines.model.pa_date_parameters import PADateParameters
+from fds.analyticsapi.engines.model.pa_identifier import PAIdentifier
+from fds.protobuf.stach.extensions.StachVersion import StachVersion
+from fds.protobuf.stach.extensions.StachExtensionFactory import StachExtensionFactory
+
+from urllib3 import Retry
+
+host = "https://api.factset.com"
+username = os.environ["ANALYTICS_API_USERNAME_SERIAL"]
+password = os.environ["ANALYTICS_API_PASSWORD"]
+
+
+def main():
+    config = Configuration()
+    config.host = host
+    config.username = username
+    config.password = password
+    # add proxy and/or disable ssl verification according to your development environment
+    # config.proxy = "<proxyUrl>"
+    config.verify_ssl = False
+
+    # Setting configuration to retry api calls on http status codes of 429 and 503.
+    config.retries = Retry(total=3, status=3, status_forcelist=frozenset([429, 503]), backoff_factor=2,
+                           raise_on_status=False)
+
+    api_client = ApiClient(config)
+
+    components_api = ComponentsApi(api_client)
+
+    try:
+        pa_document_name = "PA_DOCUMENTS:DEFAULT"
+        pa_component_name = "Weights"
+        pa_component_category = "Weights / Exposures"
+        pa_benchmark_sp_50 = "BENCH:SP50"
+        pa_benchmark_r_1000 = "BENCH:R.1000"
+        startdate = "20180101"
+        enddate = "20181231"
+        frequency = "Monthly"
+
+        components = components_api.get_pa_components(pa_document_name)
+        component_summary = ComponentSummary(name=pa_component_name, category=pa_component_category)
+        component_id = [id for id in list(components.data.keys()) if components.data[id] == component_summary][0]
+        print("PA Component Id: " + component_id)
+        pa_accounts = [PAIdentifier(id=pa_benchmark_sp_50)]
+        pa_benchmarks = [PAIdentifier(id=pa_benchmark_r_1000)]
+        pa_dates = PADateParameters(startdate=startdate, enddate=enddate, frequency=frequency)
+
+        pa_calculation_parameters = {"1": PACalculationParameters(componentid=component_id, accounts=pa_accounts,
+                                                                  benchmarks=pa_benchmarks, dates=pa_dates),
+                                     "2": PACalculationParameters(componentid=component_id, accounts=pa_accounts,
+                                                                  benchmarks=pa_benchmarks, dates=pa_dates)}
+
+        pa_calculation_parameter_root = PACalculationParametersRoot(data=pa_calculation_parameters)
```

Comment:
Can we add meta value here as they are configurable by the client?

Reply:
I think it makes sense to skip setting the meta values as we would want clients to use the defaults from our code. If we have done this in Java can we remove those values?

Reply:
Agrees. Let's remove it.